README.md

```diff
@@ -76,6 +77,28 @@ var app = new EmberApp(defaults, {
 
 The function receives the `filePath` for each asset and must return a string.
 
+### Using With `broccoli-asset-rev`
```

Comment:
Given that `broccoli-asset-rev` ships with ember-cli apps by default, I feel strongly that this belongs in the README in step-by-step fashion unless we can make the steps unnecessary. This was the biggest stumbling block for me in trying to adopt this addon after seeing your talk.